### ISSUE TITLE
Improvement points

### ISSUE THREAD

**Issue by kristijorgji** [fernan542/fl_country_code_picker#5]
Hi, first I want to congratulate you on this great library.

Previously I was using an older one https://pub.dev/packages/country_picker that had some great features that your library might benefit as well.

1. Factory to create countrycode
`CountryCode.fromCountryCode(selectionItem.value!).localize(context)`

2. Localization is not working for me.
I am changing the app locale in runtime and the picker language is still not updated

3. Allow to have no favorite icon. This is super useful and supported by the other library, you have no icon there but an elegant divider between the favorites on top and the remaining options

4. add kosovo country

5. expose `flagUri` in the country code instead of the flagImage  `Widget` type. Why ? Because the user might want to render it differently, like in the previous library you can get the Uri of the svg and render yourself for instance
```
Image.asset(
      countryInfo!.flagUri!,
      package: 'country_code_picker',
      width: 32,
    );
```

**Comment by fernan542** [comment]
Hello 👋. I'll try to implement this. 
I'm looking forward to this week.

Thanks for suggestion!

**Comment by fernan542** [comment]
This issues are now addressed at commit [9f255cc](https://github.com/fernan542/fl_country_code_picker/commit/9f255cc49a56c01b0d67d74ca530c7e6d41183ab).

Last thing, I'll just update the CHANGELOG and Readme files.
Thank you!

**Comment by fernan542** [comment]
https://pub.dev/packages/fl_country_code_picker
Update v0.1.0 🥳

* feat: add support for localization
* feat: exposed the flagUri for flag image customization
* feat: add visibility flag for favorite icon
* refactor: enable customization for modal's title
* refactor: add Kosovo country

Thank you for using my package! 😊
